internal/lookout/repository/schema/007_job_states.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE job ADD COLUMN state smallint NULL;
```

Comment:
It might be worth to set correct state for currently existing jobs as part of the migration, unless it is supper hard. You could then make it not null.